README.md

```diff
@@ -22,6 +22,9 @@ https://anuvu.github.io/zot/
 * Storage optimizations:
   * Automatic garbage collection of orphaned blobs
   * Layer deduplication using hard links when content is identical
+* Multiple storage backend using single zot server:
```

Comment:
Just say,
* Serve [multiple storage paths](example-config) (and backends) using a single zot server

Reply:
Sure